modules/tiff/src/typings/geotiff.ts

```diff
@@ -0,0 +1,74 @@
+declare module 'geotiff' {
```

Comment:
Would geotiff.js upstream be interested in a PR to add this to a `.d.ts` file? I didn't see a relevant issue.

Reply:
I'm not sure. The types are relatively incomplete (there is a much larger un-typed API). Maybe worth opening an issue.

Reply:
https://github.com/geotiffjs/geotiff.js/issues/225